piton/tools/src/sims/sims,2.0

```diff
@@ -2899,7 +2905,9 @@ sub parse_args
 
   #
   push (@{$opt{sim_run_args}}, "-gui") if ($opt{gui}) ;
+  push (@{$opt{vcs_build_args}}, "-assert svaext") ;
```

Comment:
Is this needed or can it go under a flag?

Reply:
It's needed for some basejump modules. Fine with putting it under a BlackParrot specific one, but not likely to cause issues